v2/SRD/monsters.json

```diff
@@ -59,6 +59,7 @@
             "experience": 5900,
             "hitPoints": "135 (18d10 + 36)",
             "languages": "Deep Speech, Telepathy 120 Ft.",
+            "maxHitPoints": 135,
```

Comment:
Not to nitpick, but this isn't really the max HP, as a DM could scale the creature to whatever they want or you can roll above this amount.
Would it be better to call this `hitPoints` and rename the old field `hitPointsSummary` because that's really what it is. It displays what the average hp is, the hit dice, and the static hp mod.

Thoughts?

Reply:
Technically it is the max hit points from the app's perspective. It's the max to which the monster/companion's health bar can go. 

I only called it what the Companions PR calls it. We would need to migrate both core and backend if we rename. We can do that. Agree it's not super clear.

Reply:
core and API changes are still in PR, so I think they're eligible for changes if we decided to do so.

Do you have an alternative to the naming? Or just want to keep the same?

Reply:
The more I think about it, the more I prefer `maxHitPoints`. It doesn't necessarily represent the max that the monster could have for its type, but it does represent the max hit points that the given instance of that monster would have and it makes sense to me that the HP bar goes from `maxHitPoints` to `0`. It also is consistent with the same field on the player side.

Reply:
good reason. makes sense.